.github/workflows/ci.yml

```diff
@@ -68,3 +68,7 @@ jobs:
         run: sudo apt install -y dotnet6
       - name: Run script to make sure all files end with EOL 
         run: dotnet fsi eofConvention.fsx
+      - name: Install prettier
+        run: npm install prettier@2.8.3
+      - name: Run "prettier" to style our TypeScript code
```

Comment:
@realmarv yes this step runs prettier, but it's not to style our TypeScript code, it is to check if the person that has committed his work has run prettier before committing

Reply:
Done